include/linux/soundwire/sdw_intel.h

```diff
@@ -123,4 +123,18 @@ void sdw_intel_exit(struct sdw_intel_ctx *ctx);
 
 void sdw_intel_enable_irq(void __iomem *mmio_base, bool enable);
 
+irqreturn_t sdw_intel_thread(int irq, void *dev_id);
+
+static inline
+bool is_sdw_intel_irq(struct sdw_intel_ctx *ctx)
+{
+	u32 int_status;
+
+	int_status = readl(ctx->mmio_base + 0x14);
```

Comment:
HDA_DSP_REG_ADSPIS2

Reply:
@plbossart HDA_DSP_REG_ADSPIS2 is defined at sound/soc/sof/intel/hda.h, but we don't include hda.h in soundwire driver. I am not sure include hda.h in sdw_intel.h is a good idea or not. If we just add the HDA_DSP_REG_ADSPIS2 define in sdw_intel.h, we will see redefine warning in compiling. Or we can define another name for HDA_DSP_REG_ADSPIS2 in sdw_intel.h?

Reply:
just redefine them SDW_ADSPIC2 and SDW_ADSIS2. that's fine.